### ISSUE TITLE
Error running clean-errors

### ISSUE THREAD

**Issue by that0n3guy** [riasvdv/statamic-redirect#17]
Our redirects were all down today.  We also couldn't get to our dashboard page on the cp.

I went to run `php please redirect:clean-errors` and got this error:

```
runcloud@plcom-ubuntu-s-1vcpu-1gb-nyc3-01:~/webapps/app-redacted-v3-dev$ php please redirect:clean-errors
Cleaning errors older than 1 week

   Statamic\Yaml\ParseException 

  Unable to parse at line 812 (near "9; SM-G955F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Mobile Safari/537.36'").

  at storage/redirect/errors/8d6aa1c3-6541-42d0-aab0-5c1dfbd47aa7.yaml:812
    808| hitsCount: 134
    809| latest: 1624381447
    810| handled: true
    811| handledDestination: /disability-insurance
  > 812| 9; SM-G955F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Mobile Safari/537.36'
    813|       ip: 40.94.36.61
    814|       referer: null
    815| hitsCount: 135
    816| latest: 1624381447

  • Invalid YAML: Correct any syntax errors. You may have used YAML 1.0 syntax, but 1.2 is expected. 
    https://statamic.dev/yaml

      +39 vendor frames 
  40  please:37
      Illuminate\Foundation\Console\Kernel::handle()
```

I needed things up ASAP so I just deleted the yaml file instead of messing w/ it.

We are on version 1.4.2

**Comment by riasvdv** [comment]
Seems like the user agent broke the yaml file. I'll see if I can reproduce it soon

**Comment by riasvdv** [comment]
Fixed in 1.5.0!
